slicops/auth_api.py

```diff
@@ -0,0 +1,16 @@
+"""Auth API server implementation
+
+:copyright: Copyright (c) 2024 The Board of Trustees of the Leland Stanford Junior University, through SLAC National Accelerator Laboratory (subject to receipt of any required approvals from the U.S. Dept. of Energy).  All Rights Reserved.
+:license: http://github.com/slaclab/slicops/LICENSE
+"""
+
+from pykern.pkcollections import PKDict
+from pykern.pkdebug import pkdc, pkdlog, pkdp
+import pykern.http
+import slicops.config
+
+
+class AuthAPI(pykern.http.AuthAPI):
+    """Placeholder at the moment"""
```

Comment:
This depends on the pykern 539-http-protocol branch - so maybe that should be merged before this PR.

Reply:
Yes, sorry for not documenting the dependency.